### PR TITLE
tiem: add all tiem v1.0.2 docs to master

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
tie: add all tiem docs to master

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiem/tiem-faq.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+TiEM 首次部署后，需要对 TiDB 组件规格、数据中心 Zone 信息、主机规格信息进行初始化。
+
+### 如何在命令行查看 EM 服务和 EM 部署的 TiDB 集群信息？
+
+首先请先通过 `su - tidb` 命令切换到 `tidb` 账号下。
+
+所有与 EM 工具相关的命令，请以 `TIUP_HOME=/home/tidb/.em tiup em <cmd>` 的方式执行，例如 `TIUP_HOME=/home/tidb/.em tiup em list`。
+
+所有与 EM 部署的 TiDB 集群相关的命令，请以直接以 `tiup cluster <cmd>` 的方式执行，例如 `tiup cluster list`。
```
Comment (TomShawn) [inline tiem/tiem-faq.md:62]:
```suggestion
所有与 EM 部署的 TiDB 集群相关的命令，请以直接以 `tiup cluster <cmd>` 的方式执行，例如 `tiup cluster list`。 
```

File: tiflash/tiflash-configuration.md [hunk 2]
```diff
@@ -187,6 +187,8 @@ delta_index_cache_size = 0
     # TiFlash 存储引擎的压缩级别，默认为 1。如果 dt_compression_method 设置为 LZ4，推荐将该值设为 1；如果 dt_compression_method 设置为 zstd ，推荐将该值设为 -1 或 1，设置为 -1 的压缩率更小，但是读性能会更好；如果 dt_compression_method 设置为 LZ4HC，推荐将该值设为 9。
     dt_compression_level = 1
 
+    ## 从 v5.4.0 引入，表示触发 TiFlash PageFile（log 类型） 进行 hardlink 时最小的 valid rate 值，取值范围为 [0.1,1.0]。取值越小，hardlink 越频繁；取值越大，触发 hardlink 条件越苛刻。默认为 2.0，即不开启 PageFile 的 hardlink 功能。如果 GC 时间过长，建议将该值修改成 0.8 以开启 hardlink 功能。
+    dt_storage_pool_log_gc_force_hardlink_rate = 2.0
```
Comment (TomShawn) [inline tiflash/tiflash-configuration.md:191]:
```suggestion
```

File: TOC.md [hunk 3]
```diff
@@ -490,6 +490,22 @@
...
+      - [管理集群资源](/tiem/tiem-manage-host-resources.md)
+      - [管理集群](/tiem/tiem-manage-clusters.md)
+      - [导入与导出数据](/tiem/tiem-import-and-export-data.md)
+      - [管理任务](/tiem/tiem-manage-tasks.md)
+      - [管理系统](/tiem/tiem-manage-system.md)
+    - [FAQ](/tiem/tiem-faq.md)
+    - [发布版本历史](/tiem/tiem-release-notes.md)
+      - [v1.0.0](/tiem/tiem-release-1.0.0.md)
+      - [v1.0.1](/tiem/tiem-release-1.0.1.md)
+      - [v1.0.2](/tiem/tiem-release-1.0.2.md)
```
Comment (ran-huang) [inline TOC.md:508]: 一般是把新版本放在上面

**Review by zhoubasten — APPROVED** [review]
LGTM

File: TOC.md (hunk 3 above)
Comment (Oreoxmt) [inline TOC.md]:
```suggestion
      - [v1.0.2](/tiem/tiem-release-1.0.2.md)
      - [v1.0.1](/tiem/tiem-release-1.0.1.md)
      - [v1.0.0](/tiem/tiem-release-1.0.0.md)
```

**Review by ran-huang — APPROVED** [review]
Rest LGTM

File: TOC.md (hunk 3 above)
Comment (TomShawn) [inline TOC.md:508]:
```suggestion
      - [v1.0.2](/tiem/tiem-release-1.0.2.md)
      - [v1.0.1](/tiem/tiem-release-1.0.1.md)
      - [v1.0.0](/tiem/tiem-release-1.0.0.md)
```

**Review by haiboumich — APPROVED** [review]
LGTM

File: tiem/tiem-release-1.0.2.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+# TiDB Enterprise Manager v1.0.2 Release Notes
+
+发布日期：2022 年 x 月 x 日
+
+发布版本：1.0.2
+
+TiDB Enterprise Manager (TiEM) 自 v1.0.2 起正式开放源代码。在 [v1.0.1](/tiem/tiem-release-1.0.1.md) 版的基础上，v1.0.2 有如下更新：
+
+## TiEM
```
Comment (TomShawn) [inline tiem/tiem-release-1.0.2.md:13]:
```suggestion
```

File: tiem/tiem-faq.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+
+TiEM 使用 TiDB 及其生态工具、API 提供数据库管理产品功能，TiUP 是 TiEM 使用的工具之一。TiEM 产品架构图如下：
+
+![TiEM 架构图](/media/tiem/tiem-architecture.png)
+
+## 主机资源管理 FAQ
+
+### 何时导入主机资源到 TiEM？
+
+在创建新集群或导入现有集群前，你需要将集群所需的主机资源导入到 TiEM。具体步骤参见 [TiEM v1.0.0 用户手册 - 5.3 资源管理](/tiem/tiem-manage-host-resources.md)。
```
Comment (TomShawn) [inline tiem/tiem-faq.md:34]:
```suggestion
在创建新集群或导入现有集群前，你需要将集群所需的主机资源导入到 TiEM。具体步骤参见 [TiEM 资源管理](/tiem/tiem-manage-host-resources.md)。
```

File: tiem/tiem-login-and-initialize.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+## 登录 TiEM 控制台
+
+使用 TiEM 进行操作前，你需要先登录 TiEM 控制台。
+
+要登录 TiEM 控制台，你需要先参考 [TiEM 安装手册](/tiem/tiem-install-and-maintain.md) 开启 TiEM 服务。
+
+> **注意事项：**
+>
+> TiEM v1.0.0 版本暂不支持找回密码。
```
Comment (TomShawn) [inline tiem/tiem-login-and-initialize.md:18]:
```suggestion
> TiEM v1.0.0 版本不支持找回密码。
```

File: tiem/tiem-import-and-export-data.md [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+
+TiEM 共享存储是用户事先设定好、用于保存导入导出文件的文件资源池，该文件资源池是一个可挂载于 TiEM 中控机的 NFS 目录，你可在安装 TiEM 后设置该路径。
+
+### 导入前准备
+
+进行操作前，确保已经登录 TiEM 控制台。
+
+> **注意：**
+>
+> - 从本地上传的源数据文件大小不能超过 2GB
```
Comment (Oreoxmt) [inline tiem/tiem-import-and-export-data.md:22]:
```suggestion
> - 从本地上传的源数据文件大小不能超过 2 GiB
```

File: tiem/tiem-manage-host-resources.md [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+| Kernel          | 内核版本                                                     |
+| vCPU            | CPU 核数                                                     |
+| Memory          | 内存大小                                                     |
+| NIC             | 网卡规格                                                     |
+| Cluster Purpose | 区分主机用来部署什么类型的集群，包括 TiDB、DM、TiEM          |
+| Host Purpose    | 主机用途：用来区分部署的组件类型，包括：Compute、Storage、Schedule 三种用途。多种用途以逗号 “,” 连接，例如 ‘Compute,Storage,Schedule’ |
+| Disk Type       | 磁盘类型，包括：NVMe SSD、SSD、SATA 三种类型                 |
+| Disks           | 磁盘信息，包括磁盘名称、容量、状态、路径，示例：{"name": "sda","capacity": 256,"status": "Available", "path": "/mnt/sda"} |
+
+如果通过用户名密钥方式导入主机，需要在 TiEM 安装前配置用户名和密钥路径，参见 [TiEM v1.0.0 安装和运维手册 - 指定 TiEM 中控机登录 TiDB 资源机的账户和密钥](/tiem/tiem-install-and-maintain.md#指定-tiem-中控机登录-tidb-资源机的帐户和密钥)。
```
Comment (TomShawn) [inline tiem/tiem-manage-host-resources.md:86]:
```suggestion
如果通过用户名密钥方式导入主机，需要在 TiEM 安装前配置用户名和密钥路径，参见[指定 TiEM 中控机登录 TiDB 资源机的账户和密钥](/tiem/tiem-install-and-maintain.md#指定-tiem-中控机登录-tidb-资源机的帐户和密钥)。
```

File: tiem/tiem-import-and-export-data.md [hunk 9]
```diff
@@ -0,0 +1,140 @@
...
+
+    ```shell
+    curl -X POST "http://172.16.6.206:4100/api/v1/config/update" -d "{ \"configKey\": \"ImportShareStoragePath\", \"configValue\": \"newValue\"}" -H "Authorization: Bearer 6ea768e4-c0ec-4d48-b401-0f114ddc994c"
+    ```
+
+### 导入数据操作步骤
+
+1. 进入**集群管理** > **导入导出**页面。
+2. 点击**导入数据**按钮，进入导入页面。
+3. 输入目标集群信息：集群 ID 、数据库用户名、数据库密码。
```
Comment (Oreoxmt) [inline tiem/tiem-import-and-export-data.md:66]:
```suggestion
3. 输入目标集群信息：集群 ID、数据库用户名、数据库密码。
```

File: tiem/tiem-import-and-export-data.md [hunk 10]
```diff
@@ -0,0 +1,140 @@
...
+对于保存在 TiEM 共享存储上的导入数据，你可进一步下载至本地。
+
+### 导出前准备
+
+在导出数据前，确保已经登录 TiEM 控制台。
+
+> **注意：**
+>
+> * TiEM 最多支持 3 个导入导出任务并行运行，后续任务在队列中等待。
+> * 从 TiEM 共享存储下载的导出数据文件大小不能超过 2GB。
```
Comment (Oreoxmt) [inline tiem/tiem-import-and-export-data.md:87]:
```suggestion
> * 从 TiEM 共享存储下载的导出数据文件大小不能超过 2 GiB。
```

File: tiem/tiem-import-and-export-data.md [hunk 11]
```diff
@@ -0,0 +1,140 @@
...
+
+    ```shell
+    curl -X POST "http://172.16.6.206:4100/api/v1/config/update" -d "{ \"configKey\": \"ExportShareStoragePath\", \"configValue\": \"newValue\"}" -H "Authorization: Bearer 6ea768e4-c0ec-4d48-b401-0f114ddc994c"
+    ```
+
+### 导出数据操作步骤
+
+1. 进入**集群管理** > **导入导出**页面。
+2. 点击**导出数据**按钮，进入导出数据页面。
+3. 输入源集群信息：集群 ID 、数据库用户名、数据库密码。
```
Comment (Oreoxmt) [inline tiem/tiem-import-and-export-data.md:123]:
```suggestion
3. 输入源集群信息：集群 ID、数据库用户名、数据库密码。
```

File: tiem/tiem-quick-start.md [hunk 12]
```diff
@@ -0,0 +1,162 @@
...
+| Kernel          | 内核版本                                                     |
+| vCPU            | CPU 核数                                                     |
+| Memory          | 内存大小                                                     |
+| NIC             | 网卡规格                                                     |
+| Cluster Purpose | 区分主机用来部署什么类型的集群，包括 TiDB、DM、TiEM          |
+| Host Purpose    | 主机用途：用来区分部署的组件类型，包括：Compute、Storage、Schedule 三种用途。多种用途以逗号 “,” 连接，例如 ‘Compute,Storage,Schedule’ |
+| Disk Type       | 磁盘类型，包括：NVMe SSD、SSD、SATA 三种类型                 |
+| Disks           | 磁盘信息，包括磁盘名称、容量、状态、路径，示例：{"name": "sda","capacity": 256,"status": "Available", "path": "/mnt/sda"} |
+
+如果通过用户名密钥方式导入主机，需要在 TiEM 安装前配置用户名和密钥路径，参见 [TiEM v1.0.0 安装和运维手册 - 指定 TiEM 中控机登录 TiDB 资源机的账户和密钥](/tiem/tiem-install-and-maintain.md#指定-tiem-中控机登录-tidb-资源机的帐户和密钥)。
```
Comment (TomShawn) [inline tiem/tiem-quick-start.md:88]:
```suggestion
如果通过用户名密钥方式导入主机，需要在 TiEM 安装前配置用户名和密钥路径，参见[指定 TiEM 中控机登录 TiDB 资源机的账户和密钥](/tiem/tiem-install-and-maintain.md#指定-tiem-中控机登录-tidb-资源机的帐户和密钥)。
```

File: tiem/tiem-install-and-maintain.md [hunk 13]
```diff
@@ -0,0 +1,507 @@
...
+
+## 软硬件环境配置
+
+本节介绍部署 TiEM 前需要满足的软硬件环境配置要求。
+
+### Linux 操作系统版本
+
+要部署和运行 TiEM 服务，确保 Linux 操作系统的版本满足以下要求：
+
+| Linux 操作系统平台 | 版本 |
```
Comment (Oreoxmt) [inline tiem/tiem-install-and-maintain.md:18]:
```suggestion
| Linux 操作系统 | 版本 |
```

File: tiem/tiem-install-and-maintain.md [hunk 14]
```diff
@@ -0,0 +1,507 @@
...
+| ---- | ------ | ------- | -------- | -------------------- | -------------------- |
+| TiEM | 48 核+ | 128 GB+ | SAS/SSD  | 万兆网卡（2 块最佳） | 1                    |
+
+### 网络要求
+
+TiEM 正常运行需要网络环境提供如下端口配置，管理员可根据实际环境中 TiEM 组件部署的方案，在网络侧和主机侧开放相关端口：
+
+| 组件       | 默认端口     | 说明                                   |
+| ---------- | ------------ | -------------------------------------- |
+| Web server | 4180 或 4183 | HTTP 端口：4180<br />HTTPS 端口： 4183 |
```
Comment (Oreoxmt) [inline tiem/tiem-install-and-maintain.md:58]:
```suggestion
| Web server | 4180 或 4183 | HTTP 端口：4180<br />HTTPS 端口：4183 |
```

File: tiem/tiem-install-and-maintain.md [hunk 15]
```diff
@@ -0,0 +1,507 @@
...
+| OpenAPI server| 4100 或 4103 | OpenAPI 服务端口: 4100<br />OpenAPI 监控端口: 4103 |
```
Comment (Oreoxmt) [inline tiem/tiem-install-and-maintain.md:59]:
```suggestion
| OpenAPI server| 4100 或 4103 | OpenAPI 服务端口: 4100<br />OpenAPI 监控端口：4103 |
```

File: tiem/tiem-install-and-maintain.md [hunk 16]
```diff
@@ -0,0 +1,507 @@
...
+
+> **注意：**
+>
+> 若使用旧版本浏览器或其他浏览器访问 TiEM，部分界面可能无法正常工作。
+
+## 拓扑模板
+
+在线部署 TiEM 前，你需要准备好 YAML 拓扑文件。TiEM 离线包中包含 YAML 拓扑文件模板。本节介绍用于部署 TiEM 拓扑配置模版。见 [TiEM 拓扑配置模版 config.yaml（单机版）](https://github.com/pingcap/docs-cn/blob/master/config-templates/tiem-topology-config.yaml)。
+
+如果 TiEM 中控机通过用户名密钥访问 TiDB 资源机，需要参照指定 TiEM 中控机登录 TiDB 资源机的用户名和密钥，在配置文件 config.yaml 中指定用户名和密钥。
```
Comment (Oreoxmt) [inline tiem/tiem-install-and-maintain.md:88]:
```suggestion
如果 TiEM 中控机通过用户名密钥访问 TiDB 资源机，需要参照指定 TiEM 中控机登录 TiDB 资源机的用户名和密钥，在配置文件 `config.yaml` 中指定用户名和密钥。
```

File: tiem/tiem-install-and-maintain.md [hunk 17]
```diff
@@ -0,0 +1,507 @@
...
+
+       将 TiEM v1.0.0 的 `config.yaml` 文件复制一份并覆盖 v1.0.1 的 `config-v1.0.1.yaml`。
+
+        ```shell
+        cp config.yaml config-v1.0.1.yaml
+        ```
+
+    5. 更新 `config-v1.0.1.yaml`。
+
+        覆盖配置文件后，将子步骤 3 中拷贝的 `db_path` 填写至 `config-v1.0.1.yaml`，注意空格的缩进。示例如下：。
```
Comment (Oreoxmt) [inline tiem/tiem-install-and-maintain.md:296]:
```suggestion
        覆盖配置文件后，将子步骤 3 中拷贝的 `db_path` 填写至 `config-v1.0.1.yaml`，注意空格的缩进。示例如下：
```

File: tiem/tiem-install-and-maintain.md [hunk 18]
```diff
@@ -0,0 +1,507 @@
...
+
+        ```yaml
+        em_cluster_servers:
+          - host: 127.0.0.1
+            db_path: "/home/tidb/em-backup-20220511-154415/em.db"
+        ```
+
+4. 删除 TiEM 1.0.0。
```
Comment (Oreoxmt) [inline tiem/tiem-install-and-maintain.md:304]:
```suggestion
4. 删除 TiEM v1.0.0。
```

File: tiem/tiem-overview.md [hunk 19]
```diff
@@ -0,0 +1,127 @@
...
+
+![TiEM 架构](/media/tiem/tiem-architecture.png)
+
+### Web UI
+
+TiEM Web UI 即用户可见的 TiEM 界面，展示用户输入以及服务端返回的数据。用户可与该界面进行交互，通过输入数据来获取想要看到的数据，通过界面操作完成对 TiDB 的部署运维相关工作。
+
+### OpenAPI
+
+TiEM 提供数据库管理操作的 OpenAPI，帮助用户灵活完成自动化任务或第三方集成。在 TiEM v1.0.0 中 OpenAPI 为实验特性。
```
Comment (TomShawn) [inline tiem/tiem-overview.md:26]:
```suggestion
TiEM 提供数据库管理操作的 OpenAPI，帮助用户灵活完成自动化任务或第三方集成。当前 OpenAPI 为实验特性，不建议在生产环境中使用。
```

File: tiem/tiem-install-and-maintain.md [hunk 20]
```diff
@@ -0,0 +1,507 @@
...
+    su - tidb
+
+    # 查看现有 TiEM 服务
+    TIUP_HOME=/home/tidb/.em tiup em list
+
+    # 删除指定的 TiEM 服务
+    TIUP_HOME=/home/tidb/.em tiup em destroy <cluster-name>
+    ```
+
+5. 部署 TiEM 1.0.1。
```
Comment (Oreoxmt) [inline tiem/tiem-install-and-maintain.md:319]:
```suggestion
5. 部署 TiEM v1.0.1。
```

File: tiem/tiem-install-and-maintain.md [hunk 21]
```diff
@@ -0,0 +1,507 @@
...
+
+    ```shell
+    # 切换到 tidb 账号下
+    su - tidb
+
+    # 重建资源机器 filebeat
+    TIUP_HOME=/home/tidb/.em tiup em scale-out em-test /home/tidb/em-v1.0.0-backup/resource-filebeat-scale-out-topology.yaml --user tidb -i /home/tidb/.ssh/tiup_rsa --wait-timeout 360 --yes
+    ```
+
+7. 使集群日志功能生效
```
Comment (Oreoxmt) [inline tiem/tiem-install-and-maintain.md:353]:
```suggestion
7. 使集群日志功能生效。
```

File: tiem/tiem-install-and-maintain.md [hunk 22]
```diff
@@ -0,0 +1,507 @@
...
+    从备份的元数据中恢复到新集群，流程和部署新集群相同。唯一的区别是在集群 yaml 配置中，`tiem_cluster_servers` 里增加了 `db_path: "/home/tidb/em.db"`，详细见 [TiEM 根据元数据恢复新集群拓扑配置模版 tiem.yaml（单机版）](https://github.com/pingcap/docs-cn/blob/master/config-templates/tiem-metadata-restore-config.yaml)。
+
+## 修改默认的集群备份路径
+
+TiEM 默认的集群备份路径相关配置参数如下：
+
+| 配置描述 | 配置参数名 | 参考值 |
+| --- | --- | --- |
+| TiDB 集群备份的存储类型（仅支持 NFS 或 S3） | BackupStorageType | 's3' 或 'nfs' |
+| TiDB 集群备份的存储路径 （S3 bucket 路径，或 NFS share 的绝对路径） | BackupStoragePath | 'bucketPath/backup' (S3 路径示例）<br />'/mnt/nfspath'（NFS 路径示例）|
```
Comment (Oreoxmt) [inline tiem/tiem-install-and-maintain.md:393]:
```suggestion
| TiDB 集群备份的存储路径（S3 bucket 路径，或 NFS share 的绝对路径） | BackupStoragePath | 'bucketPath/backup' (S3 路径示例）<br />'/mnt/nfspath'（NFS 路径示例）|
```

File: tiem/tiem-install-and-maintain.md [hunk 23]
```diff
@@ -0,0 +1,507 @@
...
+| TiDB 集群备份在 S3 共享存储时，S3 的 AccessKey | BackupS3AccessKey | '' |
+| TiDB 集群备份在 S3 共享存储时，S3 的 SecretAccessKey | BackupS3SecretAccessKey | '' |
+| TiDB 集群备份在 S3 共享存储时,S3 的 Endpoint（域名） | BackupS3Endpoint | '' |
```
Comment (Oreoxmt) [inline tiem/tiem-install-and-maintain.md:396]:
```suggestion
| TiDB 集群备份在 S3 共享存储时，S3 的 Endpoint（域名） | BackupS3Endpoint | '' |
```

File: tiem/tiem-install-and-maintain.md [hunk 24]
```diff
@@ -0,0 +1,507 @@
...
+    curl -X POST "http://172.16.6.206:4100/api/v1/config/update" -d "{ \"configKey\": \"BackupS3AccessKey\", \"configValue\": \"newValue\"}" -H "Authorization: Bearer 6ea768e4-c0ec-4d48-b401-0f114ddc994c"
+    ```
+
+## 修改默认的数据导入导出路径
+
+TiEM 默认的导入导出路径在 TiEM 中控机上，细节如下：
+
+| 配置描述 | 配置参数名 | 参考值 |
+| --- | --- | --- |
+| TiEM 共享存储中导入文件存储位置（建议配置为 NFS 共享存储） | ImportShareStoragePath | `/home/tiem/import`（备注：中控机 tiem 账号拥有该路径的读写权限）|
```
Comment (Oreoxmt) [inline tiem/tiem-install-and-maintain.md:434]:
```suggestion
| TiEM 共享存储中导入文件存储位置（建议配置为 NFS 共享存储） | ImportShareStoragePath | `/home/tiem/import`（备注：中控机 `tiem` 账号拥有该路径的读写权限）|
```

File: tiem/tiem-install-and-maintain.md [hunk 25]
```diff
@@ -0,0 +1,507 @@
...
+| TiEM 共享存储中导出文件存储的位置（建议配置为 NFS 共享存储） | ExportShareStoragePath | `/home/tiem/export`（备注：中控机 tiem 账号拥有该路径的读写权限）|
```
Comment (Oreoxmt) [inline tiem/tiem-install-and-maintain.md:435]:
```suggestion
| TiEM 共享存储中导出文件存储的位置（建议配置为 NFS 共享存储） | ExportShareStoragePath | `/home/tiem/export`（备注：中控机 `tiem` 账号拥有该路径的读写权限）|
```

File: tiem/tiem-install-and-maintain.md [hunk 26]
```diff
@@ -0,0 +1,507 @@
...
+
+当前 TiEM 不支持通过 TiEM 界面修改导入导出的路径。如需修导入导出路径，需要通过OpenAPI 对配置进行修改，以修改 “ImportShareStoragePath” 为例：
```
Comment (Oreoxmt) [inline tiem/tiem-install-and-maintain.md:437]:
```suggestion
当前 TiEM 不支持通过 TiEM 界面修改导入导出的路径。如需修导入导出路径，需要通过 OpenAPI 对配置进行修改，以修改 “ImportShareStoragePath” 为例：
```

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**